src/schema.ts

```diff
@@ -382,17 +338,87 @@ export interface HihatMachineSettingEvent {
 	state: HihatMachineState;
 }
 
-/** Represents the state pertaining to the hihat, not the hihat machine. */
-export interface HihatState {
-	/** Whether or not the hihat is closed. */
-	closed: boolean;
-}
+/** Any event pertaining to the hihat machine. */
+export type HihatMachineEvent = HihatMachineSettingEvent;
 
-/** Any event pertaining to the hihat, not the hihat machine. */
-export type HihatEvent = HihatClosedEvent;
 /** An event corresponding to a change in the hihat closed status. */
 export interface HihatClosedEvent {
 	kind: "hihat_closed";
 	/** Whether or not the hihat should be closed or opened. */
 	closed: boolean;
```

Comment:
I think a "closed status" is more confusing than a state where it is "open" or "closed".  (Same for having "Closed" in the interface name)

Reply:
I agree, but was overruled.